### ISSUE TITLE
 UnicodeDecodeError: 'charmap' codec can't decode byte 0x90 in position 1569: character maps to <undefined>

### ISSUE THREAD

**Issue by AKlinger1986** [cedricbonhomme/Stegano#20]
Receiving the following trying to install stegano under Python 3.6.5

I've tried re-installing and a few other things but to no avail; looks like wrong encoding in setup.py from searching around.

C:\Users\adamj\Desktop>pip install stegano
Collecting stegano
  Downloading https://files.pythonhosted.org/packages/77/76/07a61c042ac1a1cb3445689b4f140800b16d0e883a46049e221d7a92de66/Stegano-0.8.4.tar.gz (243kB)
    100% |████████████████████████████████| 245kB 3.3MB/s
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\adamj\AppData\Local\Temp\pip-install-sh3rd6wj\stegano\setup.py", line 26, in <module>
        readme = f.read()
      File "c:\python36\lib\encodings\cp1252.py", line 23, in decode
        return codecs.charmap_decode(input,self.errors,decoding_table)[0]
    UnicodeDecodeError: 'charmap' codec can't decode byte 0x90 in position 1569: character maps to <undefined>

**Comment by cedricbonhomme** [comment]
A user recently contacted me about this problem.
It was also with Python 3.6 and with Windows.

As you can see the problem is here:
https://github.com/cedricbonhomme/Stegano/blob/master/setup.py#L26
When setup.py tries to read the README file. It is because in the readme there is characters like these :
✨🐍✨ (here: https://github.com/cedricbonhomme/Stegano#installation)

The person fixed the problem by replacing the following:

```python
with open('README.rst', 'r') as f:
    readme = f.read()
with open('CHANGELOG.rst', 'r') as f:
changelog = f.read()
```

by:

```python
with open('README.rst', 'r') as f:
    readme = ''
with open('CHANGELOG.rst', 'r') as f:
    changelog = ''
```

By the way I recommend you to use the new officially recommended tool to install modules:
https://github.com/pypa/pipenv

**Comment by AKlinger1986** [comment]
Thanks for the quick response -- can this fix be provided in the base installation? I can't find the setup.py which is executed during installation (I suppose a temporary file?) to edit to get things to install.

**Comment by cedricbonhomme** [comment]
I just pushed a small changes. Can you please test it before I publish a new official release?

You can directly download the archive from the master branch:
https://github.com/cedricbonhomme/Stegano/archive/master.zip
and un-compress the file master.zip

then run:
python setup.py install
in the directory (certainly with the an admin console).

It should works. It's just a pity that there is no problem on GNU/Linus and OSX systems.

**Comment by AKlinger1986** [comment]
Great!! I was able to test that successfully with no issues

**Comment by cedricbonhomme** [comment]
Awesome, I finally manged to publish a new minor release:
https://pypi.org/project/Stegano/
You can update with pip or pipenv

**Comment by cedricbonhomme** [comment]
fixed by a3f02db66cc80b052ffe6761cf3ee49c1a46030a
